### PR TITLE
UI: Add Scans view

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
Add initial scans view with empty state, filtering and sorting. Will adjust as we get more info on mock and data from the API.

Updated title for UI as well.

### CODE REVIEW

File: client/src/components/scans/scans.js [hunk 1]
```diff
@@ -1,16 +1,187 @@
...
+    return scans.filter(item => {
+      return this.matchesFilters(item, activeFilters);
+    });
+  }
+
+  sortScans(items) {
+    const { sortType, sortAscending } = this.props;
+
+    let sortId = sortType ? sortType.id : 'name';
+    console.log(sortId);
```
Comment (cdcabrera) [inline client/src/components/scans/scans.js]: Left over ```console.log...```. Also looks like ```name``` is the default filter & sort options... both disappear they're clicked.

screen shot 2018-01-17 at 1 34 03 pm

File: client/src/components/sources/sourceConstants.js [hunk 2]
```diff
@@ -16,3 +16,23 @@ export const SourceFilterFields = [
     ]
   }
 ];
+
+const SourceSortFields = [
```
Comment (cdcabrera) [inline client/src/components/sources/sourceConstants.js]: Note: We should probably come up with some casing guidelines for bits other than classes. I've been uppercasing (yelling) my constants, may be able to add that to Prettier/Eslint

Comment (jeff-phillips-18) [reply client/src/components/sources/sourceConstants.js]: I don't mind uppercase for simple constants. Not sure I'd want to see it here.

File: client/src/components/scans/scans.js (hunk 1 above)
Comment (jeff-phillips-18) [reply client/src/components/scans/scans.js]: Fixed it up best I could for now. Treating id like name...

**Pull request merged**